tools/build-themes.sh

```diff
@@ -13,7 +13,7 @@ for FILE in $FILES
 do
   FILENAME=${FILE#$INPUT_PATH}
   BASENAME=${FILENAME%.scss}
-  $(npm bin)/node-sass $FILE > $DEST_PATH/$BASENAME.css
+  "$(npm bin)/node-sass" $FILE > $DEST_PATH/$BASENAME.css
```

Comment:
On both my dev machines, I have my source code in a directory called `Source Code` and this script wasn't a big fan of the space.  This made it work. CI should insure it works elsewhere.